src/core/lib.lua

```diff
@@ -740,6 +740,20 @@ function parse (arg, config)
    return ret
 end
 
+-- Flattens a multi-dimentional table
```

Comment:
Can you move this closer to its use please?  We have a hard time getting things into `core.lib` :)